.travis.yml

```diff
@@ -1,34 +1,13 @@
 sudo: true
 language: python
 python:
-  - "2.6"
   - "2.7"
-  - "3.4"
   - "3.5"
+  - "3.6"
 env:
-  - DJANGO=django16
-  - DJANGO=django17
-  - DJANGO=django18
```

Comment:
Given that Django 1.8 is an LTS release (scheduled for deprecation in [April 2018](https://www.djangoproject.com/weblog/2015/dec/01/django-19-released/)), I think support should be kept for it. The only thing that I think would need to be changed is something like this around the `rel` vs `remote_field` changes.

e.g.

```python
...
remote_field = 'remote_field' if django.VERSION >= (1, 9) else 'rel'
if isinstance(field, OneToOneField) and getattr(field, remote_field).parent_link:
              return True
...
```